### ISSUE TITLE
Downloader claims to offer source, but doesn't

### ISSUE THREAD

**Issue by jaylevitt** [jquerytools/www#27]
It says: "Here you can download individual tools or create a custom combination of tools by selecting only those you want. Both the source code and the minified versions are available." But, in fact, you can only get the minified versions there; you have to come here to github to get all the individual files.

**Comment by alibby251** [comment]
Hi jaylevitt,

I'm sorry - I'm not quite sure I see where the issue is? The "here" refers to the fact that source files are only available through the Github page, which is available from the download page - were you expecting the source to be available as a download via the builder?

**Comment by jaylevitt** [comment]
Yep, from the text I was expecting the builder to give me maybe a zip file with all the source.  I'd say something like "Select whichever tools like, and we'll build you a custom, minified configuration. If you want the original source, clone the [github repo](https://github.com/jquerytools/jquerytools) instead."

**Comment by alibby251** [comment]
Hi jaylevitt,

Fair play - the text isn't entirely clear on what is available and where! ;-)

But - just to confirm: the minified version(s) are only availble via the download builder; the original source is availabke through Github. You can take copies of this code or link to it directly, although the latter is not ideal, and should only really be done if you are developing something, and not for production use. The best version to use for production use is the CDN links, which link to various versions of the Tools library, and which have been minified.

I will see if we can get the wording changed, to make it clearer.

**Comment by thorinmesser** [comment]
The download page doesn't link to github.

**Comment by alibby251** [comment]
Hi thorinmesser,

The download page does actually link to Github, but perhaps its not clear - have a look for the "report a bug link on the right side of the page; this links through to Github, where you can download the source.

**Comment by windameister** [comment]
Plus, I think it is better to say:
"Or get the source code from <a>Github</a>"
than just
Or download the source code.

in the page: http://jquerytools.org/download/

It made me looking for the source code link in the whole page, still could not have one. Until I searched from google and finally got here...

Hope to be added : )
